### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
@wanda-carlson Commenting here

**Comment by wanda-carlson** [comment]
commenting here

File: Azure_70-532_Objective_1.3.htm [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <section class="answer">All Request Tracing</section>
         <section class="answer" data-correct>Failed Request Tracing</section>
         <section class="answer" data-correct>Web Server Logging</section>
+        <section class="answer" data-correct>Part2</section>
```
Comment (henry-richard) [inline Azure_70-532_Objective_1.3.htm:10]: Good changes

File: Azure_70-532_Objective_1.3.htm [hunk 2]
```diff
@@ -7,6 +7,7 @@
         <section class="answer">All Request Tracing</section>
         <section class="answer" data-correct>Failed Request Tracing</section>
         <section class="answer" data-correct>Web Server Logging</section>
+        <section class="answer" data-correct>Part2</section>
         <section class="answer">DNS Resolution Error Logging</section>
```
Comment (henry-richard) [inline Azure_70-532_Objective_1.3.htm:11]: But right
```suggestion
        <section class="answer">DNS Resolution Error Logginggggg</section>

```

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Pull request merged**